profiles/rasterbot.lua

```diff
@@ -0,0 +1,42 @@
+-- Rasterbot profile
+
+-- Minimalist node_ and way_functions in order to test source_ and segment_functions
+
+function node_function (node, result)
+end
+
+function way_function (way, result)
+  local highway = way:get_value_by_key("highway")
+  local name = way:get_value_by_key("name")
+
+  if name then
+    result.name = name
+  end
+
+  result.forward_speed = 15
+  result.backward_speed = 15
+end
+
+function source_function ()
+  raster_source = sources:load(
+    "../test/fixtures/rastersource.asc",
```

Comment:
wouldn't it be better if this path was somehow passed in as a program option when running osrm-prepare? eg --raster elevation.asc